uncollapsed_bam_processing.cwl

```diff
@@ -0,0 +1,750 @@
+class: Workflow
+cwlVersion: v1.0
+id: uncollapsed_bam_generation
+doc: >-
+  This is the workflow is written using Common Workflow Language (CWL) version
+  1.0 (https://www.commonwl.org/v1.0/) and is used at Memorial Sloan Kettering
+  Cancer Center for producing standard bam files from the NY state-approved
+  MSK-ACCESS assay. It is meant to be run on a single sample paired-end read 1
+  and read 2 fastq's, from Illumina sequencing data, but may be generalizable to
+  other sequencing platforms and sequencing panels as well, which produce
+  paired-end data.
+label: Uncollapsed BAM Generation
+$namespaces:
+  s: 'https://schema.org/'
+  sbg: 'https://www.sevenbridges.com/'
+inputs:
+  - id: input_fgbio_fastq_to_bam
```

Comment:
nitpick, but I'm wondering whether the "input_" could be removed here, to make this just `fgbio_fastq_to_bam`, or maybe leave it and call this `input_fastqs`

Reply:
although I see you have the `doc` here so maybe that's clear enough